src/sonic-dhcp-utilities/dhcp_utilities/dhcpservd/dhcp_cfggen.py

```diff
@@ -354,6 +355,9 @@ def _parse_port(self, port_ipv4, vlan_interfaces, vlan_members, ranges):
                 syslog.syslog(syslog.LOG_WARNING, f"Cannot find {splits[1]} in port_alias_map")
                 continue
             port = self.port_alias_map[splits[1]]
+            if dhcp_interface_name not in vlan_interfaces:
```

Comment:
How about move this in front of line 144?

Reply:
Current func `_parse_port` is executeed before line 144, line 368 `dhcp_interface = vlan_interfaces[dhcp_interface_name]` would encounter KeyError without this check.